projects/1.16/assets/ticbc/bettercompat/lang/zh_cn.json

```
@@ -0,0 +1,259 @@
+"modifier.bettercompat.barterable": "交易",
+"modifier.bettercompat.barterable.flavor": "财富!",
+"modifier.bettercompat.barterable.description": "降低所有村民交易的价格",
+"modifier.bettercompat.enderference": "末地参照",
+"modifier.bettercompat.enderference.flavor": "末地情报!",
+"modifier.bettercompat.enderference.description": "防止末影人在攻击时瞬移",
+"modifier.bettercompat.quartz_infused": "石英注入",
+"modifier.bettercompat.quartz_infused.flavor": "时间!",
+"modifier.bettercompat.quartz_infused.description": "石英注入会略微增加伤害",
+"modifier.bettercompat.lava_efficiency": "岩浆效率",
+"modifier.bettercompat.lava_efficiency.flavor": "熔岩!",
+"modifier.bettercompat.lava_efficiency.description": "使用者在岩浆中的采矿速度更快并可以造成更多伤害",
+"modifier.bettercompat.crystal_power": "水晶之力",
+"modifier.bettercompat.crystal_power.flavor": "水晶洞!",
+"modifier.bettercompat.crystal_power.description": "提高受影响材料的采矿速度",
+"modifier.bettercompat.moonlight": "月光",
+"modifier.bettercompat.moonlight.flavor": "月亮!",
+"modifier.bettercompat.moonlight.description": "在夜晚增强你的工具",
+"modifier.bettercompat.elemental": "元素之力",
+"modifier.bettercompat.elemental.flavor": "元素!",
+"modifier.bettercompat.elemental.description": "攻击一个怪物后给你随机的元素效果",
+"modifier.bettercompat.inclined": "倾斜",
+"modifier.bettercompat.inclined.flavor": "高耸入云!",
+"modifier.bettercompat.inclined.description": "你的海拔越高，你的工具就越强大",
+"modifier.bettercompat.dark_might": "黑暗之力",
+"modifier.bettercompat.dark_might.flavor": "胆大包天!",
+"modifier.bettercompat.dark_might.description": "如果你在和特殊的生物战斗，你可能会进行反击",
+"modifier.bettercompat.fiery": "灼燃",
+"modifier.bettercompat.fiery.flavor": "刺激!",
+"modifier.bettercompat.fiery.description": "利用火龙的力量向你的对手开火",
+"modifier.bettercompat.icy": "冰霜",
+"modifier.bettercompat.icy.flavor": "极寒!",
+"modifier.bettercompat.icy.description": "利用冰龙的力量冻结你的对手",
+"modifier.bettercompat.electric": "雷击",
+"modifier.bettercompat.electric.flavor": "震撼!",
+"modifier.bettercompat.electric.description": "利用电龙的力量电击你的对手",
+"modifier.bettercompat.ender_crushing": "末影冲击",
+"modifier.bettercompat.ender_crushing.flavor": "了不起!",
+"modifier.bettercompat.ender_crushing.description": "使用终界的力量来增加你开采末地石的能力",
+"modifier.bettercompat.floras_blessing": "弗洛拉的祝福",
+"modifier.bettercompat.floras_blessing.flavor": "自然之力!",
+"modifier.bettercompat.floras_blessing.description": "瞬间破坏一个方块或杀死一个生物",
+"modifier.bettercompat.holy": "神圣",
+"modifier.bettercompat.holy.flavor": "祝福之刃!",
+"modifier.bettercompat.holy.description": "让亡灵生物倒在你的刀下",
+"modifier.bettercompat.mana": "魔力",
+"modifier.bettercompat.mana.flavor": "如有神助!",
+"modifier.bettercompat.mana.description": "当库存中有魔力平板时，使用魔力来修复你的工具",
+"modifier.bettercompat.clime": "气候",
+"modifier.bettercompat.clime.flavor": "天气的力量!",
+"modifier.bettercompat.clime.description": "温度越低，受到的击退越弱",
+"modifier.bettercompat.obsidianified": "Obsidianified",
+"modifier.bettercompat.obsidianified.flavor": "极限抵抗!",
+"modifier.bettercompat.obsidianified.description": "根据流体，水或熔岩，你的工具效率会有所不同。也坚不可摧！",
```

```suggestion
"modifier.bettercompat.obsidianified": "黑耀化",
"modifier.bettercompat.obsidianified.flavor": "极限抵抗！",
"modifier.bettercompat.obsidianified.description": "根据身处的流体（水或熔岩），你的工具效率会有所不同。同时工具坚不可摧！",
```
源码：
持有者眼部位于水中时，采掘速度 = 原采掘速度 \* (属性等级 * 1.5)
持有者眼部位于熔岩中时，采掘速度 = 原采掘速度 -  (属性等级 * 1.5)
其余情况没有加成